source/_integrations/reolink.markdown

```diff
@@ -476,6 +476,128 @@ For example, you can place arrow buttons on the card to [control the camera](/da
 
 You can receive rich notifications on your phone when someone rings a Reolink doorbell or a Reolink camera detects an event like motion or a person.
 
+<p class='img'>
+  <img src='/images/integrations/reolink/rich_notification__small_notification.jpg' alt='Screenshot: Small phone notification'>
+  <img src='/images/integrations/reolink/rich_notification__big_notification.jpg' alt='Screenshot: Expanded phone notification'>
+</p>
+
+{% details "Rich notification tutorial" icon="mdi:cursor-hand" %}
+
+Prerequisites:
+
+- This [Reolink integration](#configuration)
+- [Android or iOS companion app](https://companion.home-assistant.io/docs/getting_started#setting-up)
+- [Remote access to Home Assistant](https://www.home-assistant.io/docs/configuration/remote/). Although you can receive text notifications without remote access, to see the camera image in the notification (rich notification), the phone needs to be able to reach Home Assistant. The rich notification will always work, even without remote access, when the phone is on the same network as Home Assistant.
+
+1. In order to receive such a rich notification, we are going to make an automation in Home Assistant. In Home Assistant go to {% my automations title="**Settings** > **Automations & scenes**" %} and select **Create automation** > **Create new automation**.
+
+![Settings button](/images/integrations/reolink/rich_notification__settings.png)
+![Automations & scenes button](/images/integrations/reolink/rich_notification__automations_and_scenes.png)
+![Create automation button](/images/integrations/reolink/rich_notification__create_automation.png)
+
```

Comment:
I think there are indents missing beneath the steps. as in, the images and substeps should have an indent.
![image](https://github.com/user-attachments/assets/4aba760e-5882-4ba5-82f7-63ef789ce85f)

in cases where there is a substep, I would use a bullet and an indent, to make it clear that it is a substep.

I would make suggestions, but I might mess the indents up here in GH. Probably easier to get it right in a proper editor.

Reply:
@c0ffeeca7 how do you actually make a indent?
Just adding spaces apparently does not work...